Bio/SeqAnalysis/analyzer_tests.py

```diff
@@ -0,0 +1,3 @@
+from Bio.SeqRecord import  SeqRecord
+seq_analyzer = Bio.SeqAnalysis.analyzer(records)
+reults = seq_analyzer.results()
```

Comment:
Wrong tests. Tests should check if output from predetermined input is correct. This test does check nothing.

Also, is it even run?

Reply:
No, it wont. Work still in porgress xd

Reply:
Also am i supossed to write a test for my class (SeqAnalyzer) as a whole or for each funcions separately?

Reply:
Ok :D in that cases, please add some proper comment like:
```
# TODO: fix tests, currently not working
```

Reply:
> Also am i supossed to write a test for my class (SeqAnalyzer) as a whole or for each funcions separately?

Should be both, unit tests are for each function, integration test are for whole thing. If you don't want to do this in this pull request, you can just add comment like above and add tests later.